packages/react/src/themes/teams-high-contrast/components/Chat/chatMessageVariables.ts

```diff
@@ -7,7 +7,7 @@ export default (siteVars: any): Partial<ChatMessageVariables> => {
     authorColor: siteVars.colors.white,
     contentColor: siteVars.colors.white,
     color: siteVars.colors.white,
-    contentFocusOutlineColor: siteVars.colors.yellow[900], // Red flag (should this be accessibleYellow?)
+    contentColorFocus: siteVars.colors.yellow[900], // Red flag (should this be accessibleYellow?)
```

Comment:
I am pretty sure this should be accessibleYellow, can we update it?

Reply:
done, thanks